### DISCUSSION TITLE
COPY failed: file not found in build context or excluded by .dockerignore: stat yarn.lock: file does not exist

### DISCUSSION THREAD

**Discussion by allcho** [vercel/next.js#33338] [Help]
Hello,  try to using Docker

I do build container: docker build -t nextjs-docker . and get failed

**Sending build context to Docker daemon  219.1kB
Step 1/23 : FROM node:16-alpine AS deps
 ---> 23990429c0d7
Step 2/23 : RUN apk add --no-cache libc6-compat
 ---> Using cache
 ---> 7544c5d49f28
Step 3/23 : WORKDIR /app
 ---> Using cache
 ---> a9b3ee4350bc
Step 4/23 : COPY package.json yarn.lock ./
COPY failed: file not found in build context or excluded by .dockerignore: stat yarn.lock: file does not exist**

Whats wrog I do?

**Comment by GabenGar** [comment]
Do you have `yarn.lock` in the source directory?
